docs/Contributing.asciidoc

```diff
@@ -656,39 +645,17 @@ So by replacing VAR1 and VAR2 with those values one can trigger the different te
 Of course it is also possible to run (specific) tests directly via `prove` instead of using the Makefile targets.
 
 ==== Tips
-Commands passed to `podman run` will be executed after the initialization script (which does database creation and so on). So if there is
-the need to run an interactive session after it just do:
-
-  podman run -it -v OPENQA_LOCAL_CODE:/opt/openqa openqa:latest bash
-
-Of course you can also use `make run-tests-within-container \; bash` to run the tests first and then open a shell for further investigation.
-
-There is also the possibility to change the initialization scripts with the `--entrypoint switch`. This allows us to go into an interactive
-session without any initialization script run:
-
-  podman run -it --entrypoint /bin/bash -v OPENQA_LOCAL_CODE:/opt/openqa registry.opensuse.org/devel/openqa/containers/openqa_dev
-
-In case there is the need to follow what is happening in the currently running container (the execution will terminate the session):
-
-  podman exec -ti $(podman ps | awk '!/CONTAINER/{print $1}') /bin/bash
 
 Running UI tests in non-headless mode is also possible, eg.:
 
-  xhost `local:root
-  podman run --rm -ti --name openqa-testsuite -v /tmp/.X11-unix:/tmp/.X11-unix:rw -e DISPLAY="$DISPLAY" -e NOT_HEADLESS=1 openqa:latest prove -v t/ui/14-dashboard.t
+  xhost +local:root
+  podman run --rm -ti --name openqa-testsuite -v /tmp/.X11-unix:/tmp/.X11-unix:rw -e DISPLAY="$DISPLAY" -e NOT_HEADLESS=1 openqa_devel:latest prove -v t/ui/14-dashboard.t
   xhost -local:root
 
 It is also possible to use a custom os-autoinst checkout using the following arguments:
 
   podman run … -e CUSTOM_OS_AUTOINST=1 -v /path/to/your/os-autoinst:/opt/os-autoinst make run-tests-within-container
 
-By default, `configure` and `make` are still executed (so a clean checkout is expected). If your checkout is already prepared to use,
-set `CUSTOM_OS_AUTOINST_SKIP_BUILD` to prevent this. Be aware that the build produced outside of the container might not work inside the
-container if both environments provide different, incompatible library versions (eg. OpenCV).
```

Comment:
The  `run-tests-within-container` script still contains the logic regarding `CUSTOM_OS_AUTOINST_SKIP_BUILD`. The remark about builds being possibly incompatible is probably also still valid.

Reply:
Ah, ok. So this is not the entrypoint script feature but `run-tests-within-container` feature. I will check if that is applicable for the new image as well and adapt/keep it then.